app/utils/callback/lava.py

```diff
@@ -601,7 +608,9 @@ def add_tests(job_data, job_meta, lab_name, db_options,
     }
 
     try:
-        _get_job_meta(meta, job_data)
+        if not _get_job_meta(meta, job_data):
+            ret_code = 400
+            msg = "Unsupported job metadata"
```

Comment:
But then, the rest still gets run even though we know we want to discard this job, right?

Reply:
Right, I was looking for a solution that wouldn't require too many changes in the current error handling solution. The other idea was raising a `ValueError` in the `_get_job_meta()`, which is not quite what should be done here, too.
None of these solutions seem 100% good in this situation.